cms/topics/models.py

```diff
@@ -86,7 +86,7 @@ class TopicPage(ExclusiveTaxonomyMixin, BasePage):  # type: ignore[django-manage
     explore_more = StreamField(ExploreMoreStoryBlock(), blank=True)
 
     headline_figures = StreamField(
-        TopicHeadlineFiguresStreamBlock(),
+        [("figure", TopicHeadlineFigureBlock())],
```

Comment:
This will need a StreamField data migration - https://docs.wagtail.org/en/stable/reference/streamfield/data_migrations.html

Reply:
Added in [7b36ee7](https://github.com/ONSdigital/dis-wagtail/pull/181/commits/7b36ee7b382f08c889969d2a64f75075ddd20e34).

None of the existing Operations did what I needed to do (tried `RenameStreamChildrenOperation` and `RenameStructChildrenOperation`) so I wrote my own one. Let me know if they _should have worked_ and I'll try to work out why they weren't working for me.

Reply:
The StreamField data migrations mechanism takes care of revisions as well, whereas 7b36ee7 doesn't. We want a similar thing. tl;dr go through each revision, load as instance make the transformation and update the revision too

Reply:
Addressed in [cc4d741](https://github.com/ONSdigital/dis-wagtail/pull/181/commits/cc4d74194702c097e850ec5d84d47349403e943a)